Calendar/Get-RBASummary.ps1

```diff
@@ -32,17 +32,26 @@ if (Test-ScriptVersion -AutoUpdate) {
 
 Write-Verbose "Script Versions: $BuildVersion"
 
+$SummaryFilename = "RBA-Summary-For_$($Identity.Split('@')[0])_$((Get-Date).ToString('yyyy-MM-dd_HH-mm-ss')).txt"
+Write-Host "`r`nRBA Summary Output saved as [" -NoNewline
+Write-Host -ForegroundColor Cyan $SummaryFilename -NoNewline
+Write-Host "] in the current directory."
+Start-Transcript -Path $SummaryFilename
```

Comment:
I never really used `Start-Transcript` but does it include `Write-Verbose` in the log output? If it doesn't, you should look at the logging function that we created in Shared.

Reply:
Start-Transcript does include Verbose output (assuming you pass in -Verbose). 
It is quick and easy, but I will look for the Logging function in Shared for future scripts.